src/github.com/couchbase/sync_gateway/db/change_cache.go

```diff
@@ -578,6 +592,18 @@ func (h *SkippedSequenceQueue) Remove(x uint64) error {
 
 }
 
+// Remove does a simple binary search to find and remove.
+func (h *SkippedSequenceQueue) Contains(x uint64) bool {
```

Comment:
Can `Contains()` be a value receiver rather than a pointer receiver?  It seems it is dereferencing the pointer in all uses of it anyway.  The value add of making it a value receiver would be easier code readability.